recipes/lora_finetune_distributed.py

```diff
@@ -151,16 +146,6 @@ def __init__(self, cfg: DictConfig) -> None:
         self._log_every_n_steps = cfg.get("log_every_n_steps", 1)
         self._log_peak_memory_stats = cfg.get("log_peak_memory_stats", False)
 
-        # training attributes
-        self._enable_activation_checkpointing = cfg.enable_activation_checkpointing
-        self._enable_activation_offloading = cfg.get(
-            "enable_activation_offloading", False
-        )
-        if self._enable_activation_offloading and self._device.type != "cuda":
-            raise RuntimeError(
-                "enable_activation_offloading should only be enabled for training on CUDA"
-            )
```

Comment:
removed from init. This is handled in setup_model

Reply:
I added it back after this:

![image](https://github.com/user-attachments/assets/bd3d8743-b850-410c-9117-b314090141f5)